_test/disabled_tests.md

```diff
@@ -7,7 +7,7 @@
   - test_labprobe_nonmpi (no ticket)
 
 - test_job_dispatcher_parpool.m
-  - test_job_submittion (no ticket)
+  - test_job_submittion (Its not a disabled test, but a tester to debug job submission on a cluster)
```

Comment:
If this isn't an actual test, we could probably remove it from here.

Reply:
may be name can be changed
The code was exteamly useful for debugging jobs on cluster so I do not want to remember how to do it when time come to run jobs on SCARF

And it is a good place to find it

Reply:
as the first thing you will be doing testing cluster, is check that the cluster config is correct

This is the test, which should be run on SCARF only. For everything else its substantial waste of time. 
When we have is_jenkins & is_scarf -- this test should run